build/commonTest.props

```diff
@@ -25,8 +25,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
[Same](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2573/files#r1582438994).